asn1tools/source/c/oer.py

```diff
@@ -526,7 +527,7 @@ def format_real(self, type_):
             return ['double']
 
     def get_enumerated_values(self, type_):
-        return type_.value_to_data.values()
+        return sorted(type_.data_to_value.items(), key=itemgetter(1))
```

Comment:
Doesn't `sorted(type_.value_to_data.values())`, or possibly `sorted(type_.data_to_value.values())`, work?

Reply:
I need the ".items()" (tuple key/value), because when declaring the enum, I need both the value and the data/name instead to only the name like before. Sorting the items without the key will yield a list sorted by enum name (which would work as well, but look a bit strange): I need the list sorted by value.

Reply:
I didn't realize that you changed the return type of the function. Now the change makes sense. =)